src/coreclr/scripts/coreclr_arguments.py

```diff
@@ -156,6 +158,20 @@ def verify(self,
     ############################################################################
 
     def __initialize__(self, args):
+
+        def provide_default_host_os():
```

Comment:
what was the goal for moving these functions? Were any errors or do we have a python code convention?

Reply:
I thought it was confusing to have the callers appear before the callees. So, there's no real convention here that I knew of, but I did rearrange things to match my (personal) convention better... So you could say this change was somewhat arbitrary.